frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/OpprettTiltaksgjennomforingContainer.tsx

```diff
@@ -323,7 +326,7 @@ export const OpprettTiltaksgjennomforingContainer = (
         orgnr: arrangorOrgnr,
       });
 
-    const lokasjonsStreng = `${postnummer} ${poststed}`;
```

Comment:
Kunne vel bare kalt `.trim()?  på streng-konkatineringen for å fjerne mellomrom når det ikke finnes en verdi der